CMakePresets.json

```diff
@@ -78,6 +78,20 @@
         "CMAKE_CXX_COMPILER": "clang++"
       }
     },
+    {
+      "name": "apple-m1-base",
```

Comment:
How about `apple-arm64-base` so that there’s no confusion when using it with M2, M3, etc

Reply:
Nice suggestion! Will fix

Reply:
Fixed